requirements.txt

```diff
@@ -34,3 +34,4 @@ starlette~=0.14.2
 Unidecode==1.2.0
 uvicorn~=0.13.4
 versioneer==0.19
+fideslog==1.1.5
```

Comment:
A very talented and good-looking engineer recently alphabetized this file in https://github.com/ethyca/fidesops/pull/491/commits/ecfb63d333daa24d613be73e6689fee2d8adc873 (#491). Please keep it nice 😄

Reply:
would be nice to have this automated via linter or some auto formatter. Thoughts?